common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -240,8 +256,12 @@ def create_event(self, knowledge_node_uid, student_uid, event_type, **data):
         payload.update(data)
 
         # Send request
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        event = json.loads(response.content)
+        event = {}
+        try:
+            response = requests.post(url, headers=self.request_headers, data=payload)
+            event = json.loads(response.content)
+        except requests.ConnectionError:
+            log.warning('Could not post create an event.')
```

Comment:
@macornwell Please update the message to say:

```python
"Could not create event of type {event_type} for unit identified by knowledge_node_uid {knowledge_node_uid} and student identified by student_uid {student_uid}.".format(event_type=event_type, student_uid=student_uid, knowledge_node_uid=knowledge_node_uid)
```

You'll need to add line breaks as necessary.

Also, let's put `event = {}` into the `except` clause (above the log statement).